vault/table_pki_cert.go

```diff
@@ -0,0 +1,87 @@
+package vault
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+// Types
+type PkiCert struct {
+	Path          string
+	Serial        string
+	RequestID     string
+	LeaseID       string
+	LeaseDuration int64
+	Renewable     bool
+}
+
+// Table Function
+func tablePkiCert() *plugin.Table {
+	return &plugin.Table{
+		Name:        "vault_pki_cert",
+		Description: "Vault PKI Certificates",
+		List: &plugin.ListConfig{
+			Hydrate: listCerts,
+		},
+		Columns: []*plugin.Column{
+			{Name: "path", Type: proto.ColumnType_STRING, Description: "The path (mount point) of the engine containing the PKI certificate"},
+			{Name: "serial", Type: proto.ColumnType_STRING, Description: "The serial identifier of the certificate"},
+			{Name: "request_id", Type: proto.ColumnType_STRING, Description: "Request Identifier"},
+			{Name: "lease_id", Type: proto.ColumnType_STRING, Description: "Lease Identifier", Transform: transform.FromGo()},
+			{Name: "lease_duration", Type: proto.ColumnType_INT, Description: "Duration of Lease in seconds (0 if not set)", Transform: transform.FromGo()},
```

Comment:
when 0, does that imply infinite? might be good to add what it means when it's "not set"